src/aics-image-viewer/components/App/index.tsx

```diff
@@ -438,6 +445,8 @@ export default class App extends React.Component<AppProps, AppState> {
     view3d.setCameraMode(enums.viewMode.VIEW_MODE_ENUM_TO_LABEL_MAP.get(userSelections.mode));
     view3d.setShowAxis(userSelections[SHOW_AXES]);
     view3d.setShowBoundingBox(aimg, userSelections.showBoundingBox);
+    view3d.setBoundingBoxColor(aimg, colorArrayToFloatArray(userSelections.boundingBoxColor));
+    view3d.setBackgroundColor(colorArrayToFloatArray(userSelections.backgroundColor));
```

Comment:
since this (bg color) doesn't depend on aimg, it probably should not be called here.